README.md

```diff
@@ -145,41 +180,55 @@ All dashboards (JSON files) are auto-loaded from directory [`grafana/dashboards/
 * Node exporter full [[source]](https://grafana.com/grafana/dashboards/1860-node-exporter-full/)
 * Loki and Promtail [[source]](https://grafana.com/grafana/dashboards/14055-loki-stack-monitoring-promtail-loki/)
 
-## Stop
 
-* If you used a blocking `docker compose -p obs up`, just cancel via keyboard with `[Ctrl]+[c]`
+## Accessing [Daml Enterprise](https://www.digitalasset.com/products/daml-enterprise) Docker Images
 
-* If you detached compose: `docker compose -p obs down`
+1. Get an access key from digital asset.  This step needs to be performeed
+  once.  If you haven't logged in before, you can get an access key by logging
+  in to [digitalasset.jfrog.io](https://digitalasset.jfrog.io) using your
+  Google account and generating an identity token on your [Artifactory profile
```

Comment:
I'm not sure if having a Google account is a requirement, i.e., you may log in to `digitalasset.jfrog.io` using a simple username/password pair as well.

Reply:
Removed reference to Google account